docs-chef-io/content/supermarket/config_rb_supermarket.md

```diff
@@ -76,6 +76,10 @@ integration. Default value: `'tools'`.
 
 : The directory that Supermarket will use to store logs. Default value: `'/var/log/supermarket'`.
 
+`default['supermarket']['log_mode']`
+
+: The Supermarket log directory permission which can be configured. Default value: `700`.
```

Comment:
```suggestion
: The Supermarket log directory file permissions. Default value: `700`.
```

Reply:
Should the default be '700' or '0700'? Compare to omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

Reply:
Updated!! Thanks @kagarmoe . Not much difference as we are supposed to pass this value to `chmod`.